### ISSUE TITLE
Fix lint issues

### ISSUE THREAD

**Issue by mewmew** [mewmew/float#4]
As reported by `golint` and `golangci-lint`: https://travis-ci.org/mewmew/floats/builds/398749776

golint:
```
floats.go:8:6: exported type Float should have comment or be unexported
floats.go:13:1: exported function New should have comment or be unexported
floats.go:19:1: exported method Float.Float64 should have comment or be unexported
binary128/binary128.go:72:17: should drop = 0 from declaration of var a; it is the zero value
```

golangci-lint:
```
$ golangci-lint run --enable-all
binary128/binary128.go:72:17: should drop = 0 from declaration of var a; it is the zero value (golint)
	var a uint64 = 0
	               ^
binary128/binary128.go:30: File is not goimports-ed (goimports)
	sign := intRep & 0x80000000 != 0
binary128/binary128_test.go:4: File is not goimports-ed (goimports)
	"testing"
binary128/binary128_test.go:43:6: `float64FromString` is unused (deadcode)
func float64FromString(s string) float64 {
     ^
binary128/binary128.go:63:4: should merge variable declaration with assignment on next line (megacheck)
			var a uint64
			^
binary128/binary128.go:77:2: should merge variable declaration with assignment on next line (megacheck)
	var newExp uint64
	^
binary128/binary128.go:81:2: should merge variable declaration with assignment on next line (megacheck)
	var newMant uint64
	^
```

**Comment by alexpantyukhin** [comment]
I will fix it :)

**Comment by mewmew** [comment]
Documentation related lint issues were fixed in rev 0807f887bd5ab5e425e5181a2848a2d888bcb65a.

```
floats.go:8:6: exported type Float should have comment or be unexported
floats.go:13:1: exported function New should have comment or be unexported
floats.go:19:1: exported method Float.Float64 should have comment or be unexported
```

**Comment by mewmew** [comment]
Fixed. Only thing left is a duplication notice for test cases. That's fine.
